subset/network/NTPClient/src/main/java/Main.java

```diff
@@ -19,6 +20,9 @@ public static void main(String[] args) {
         serverName = args[0];
         PORT = Integer.parseInt(args[1]);
         timerPeriod = Integer.parseInt(args[2]);
+        if (args.length == 4) {
```

Comment:
DOes this need to be optional?  Seems better just to make it required all the time?

Should check for the bad case (# of args != 4) and print a message.